### PR TITLE
chore(gatsby-plugin-sharp, gatsby-transformer-sharp): update dependencies

### PR DESCRIPTION
**Pull request by hoobdeebla**
## Description

Updates dependencies of gatsby-plugin-sharp and gatsby-transformer-sharp to the latest Node 10.13-compatible versions. got was updated to version 10 as got@11 is incompatible with Node <10.19. Also updates probe-image-size in gatsby-remark-copy-linked-files and sharp in gatsby-plugin-manifest and gatsby-remark-images-contentful.

### Semver-Major Updates
| dependency | old | new |
|-|-|-|
| async | `^2.6.3` | `^3.2.0` |
| fs-extra | `^8.1.0` | `^9.0.1` |
| got | `^8.3.2` | `^10.7.0` |
| imagemin | `^6.1.0` | `^7.0.1` |
| imagemin-mozjpeg | `^8.0.0` | `^9.0.0` |
| imagemin-pngquant | `^6.0.1` | `^9.0.0` |
| probe-image-size | `^4.1.1` | `^5.0.0` |

### Relevant Breaking Changes

[async:](/caolan/async/blob/master/CHANGELOG.md#v300) no API changes used by these modules
[fs-extra:](/jprichardson/node-fs-extra/blob/master/CHANGELOG.md) require Node 10; no API changes used by these modules
[got v9:](/sindresorhus/got/releases/tag/v9.0.0) require Node 8; no API changes used by these modules
[got v10:](/sindresorhus/got/releases/tag/v10.0.0) require Node 10; change the `json` option to accept an object instead of a boolean and to only be responsible for the request, not the response (packages/gatsby-plugin-sharp/src/scheduler.js#L58) - `got.post({body: {x: true}, json: true});` →  ` got.post({json: {x: true}, responseType: 'json'});`
[imagemin:](/imagemin/imagemin/releases/tag/v7.0.0) require Node 8; no API changes used by these modules
[imagemin-mozjpeg:](/imagemin/imagemin-mozjpeg/releases/tag/v9.0.0) require Node 10; no API changes
[imagemin-pngquant v7:](/imagemin/imagemin-pngquant/releases/tag/v7.0.0) change the `quality` option to accept an array of min and max elements and use fractional numbers (packages/gatsby-plugin-sharp/src/process-file.js#L197) - `{quality: '60-80'}` → `{quality: [0.6, 0.8]}`
[imagemin-pngquant v8:](/imagemin/imagemin-pngquant/releases/tag/v8.0.0) require Node 8; no API changes
[imagemin-pngquant v9:](/imagemin/imagemin-pngquant/releases/tag/v9.0.0) require Node 10; no API changes
[probe-image-size:](/nodeca/probe-image-size/blob/master/CHANGELOG.md#500---2019-09-14) no API changes

### CODE REVIEW

**Comment by hoobdeebla** [comment]
@wardpeet, anything I can do to help get this merged?

**Comment by hoobdeebla** [comment]
Hello @wardpeet, I would very much appreciate a review.

**Comment by wardpeet** [comment]
Sorry @hoobdeebla, i'll be checking it out as we speak :)

File: packages/gatsby-plugin-sharp/src/scheduler.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 const { v4: uuidv4 } = require(`uuid`)
 const path = require(`path`)
-const fs = require(`fs-extra`)
+const fs = require(`fs`)
```
Comment (wardpeet) [inline packages/gatsby-plugin-sharp/src/scheduler.js:3]: Please keep fs-extra as it has better windows support.
```suggestion
const fs = require(`fs-extra`)
```

**Review by wardpeet — CHANGES_REQUESTED** [review]
1 small remark :) looks good! thanks for taking the time

**Comment by hoobdeebla** [comment]
All fixed, all green. Thank you for the review!

**Review by wardpeet — APPROVED** [review]
LGTM! thanks for fixing and upgrading

**Pull request merged**